glean-core/android/src/main/java/mozilla/telemetry/glean/upload/Upload.kt

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.upload
+
+import org.json.JSONObject
+import com.sun.jna.Structure
+import com.sun.jna.Pointer
+import com.sun.jna.Union
+import mozilla.telemetry.glean.Glean
+import mozilla.telemetry.glean.net.HeadersList
+import mozilla.telemetry.glean.rust.getRustString
+
+@Suppress("ClassNaming")
+@Structure.FieldOrder("tag", "uuid", "path", "body", "headers")
+internal class Upload_Body(
```

Comment:
That's cbindgen names it (https://github.com/mozilla/glean/blob/m16/ping-uploading/glean-core/ffi/glean.h#L88). I tried doing it without the underscore, but the type mapping fails.

Reply:
Ok, that's fine. Please add a small comment static that.

Reply:
Side-note: It doesn't _have_ to match the generated C header file. The C header file is unused for Kotlin. Names are a compile-time thing only anyway.

Reply:
> Side-note: It doesn't have to match the generated C header file. The C header file is unused for Kotlin. Names are a compile-time thing only anyway.

That was my understanding too, when I first tried this it failed. Probably because of some other reason though, I changed it now and it worked. Underscore removed :)